apple/swift-docc-render
# Fix preview box getting cut off for Tutorial pages

Bug/issue #, if applicable: rdar://93843834

## Summary
Fixes issue where tutorial preview frame looks broken after hiding and re-expanding preview. 

## Testing
Steps:
1. Navigate to a tutorial page with previews
2. Verify that UI is consistent even after hiding and re-expanding the preview box. 

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
